### PR TITLE
Proper highlighting for triple angle bracket command blocks

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: WDL.YAML-tmLanguage [hunk 1]
```diff
@@ -9,13 +9,13 @@ patterns:
   match: \=
 
 - name: keyword.operator.comparison.wdl
-  match: <\=|>\=|\=\=|<|>|\!\=
+  match: <\=|>\=|\=\=|<{1}|>{1}|\!\=
```
Comment (cjllanwarne) [inline WDL.YAML-tmLanguage]: I'm curious whether these are needed - unless they enforce not having two in a row this seems identical to what was there before?

Comment (ghost) [reply WDL.YAML-tmLanguage]: I was being a bit overzealous! Will remove.

File: WDL.YAML-tmLanguage [hunk 2]
```diff
@@ -15,7 +15,7 @@ patterns:
   match: \+\=|-\=|\*\=|/\=|//\=|%\=|&\=|\|\=|\^\=|>>\=|<<\=|\*\*\=
 
 - name: keyword.operator.arithmetic.wdl
-  match: \+|\-|\*|\*\*|/|//|%|<<|>>|&|\||\^|~
+  match: \+|\-|\*|\*\*|/|//|%|<|>|&|\||\^|~
```
Comment (cjllanwarne) [inline WDL.YAML-tmLanguage]: Should stay as `<<` and `>>`

Comment (ghost) [reply WDL.YAML-tmLanguage]: 🤦‍♂️

**Comment by cjllanwarne** [comment]
This is great! 👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/wdl-sublime-syntax-highlighter/pull-request/12/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/wdl-sublime-syntax-highlighter)

**Pull request merged**